src/parameterized_tendencies/radiation/radiation.jl

```diff
@@ -425,12 +425,12 @@ end
 
 function radiation_model_cache(Y, radiation_mode::RadiationDYCOMS)
     FT = Spaces.undertype(axes(Y.c))
-    NT = NamedTuple{(:z, :ρ, :q_tot), NTuple{3, FT}}
+    NT = NamedTuple{(:z, :ρ, :ρq_tot), NTuple{3, FT}}
```

Comment:
Why do we need the \rho q_tot symbol in the named tuple, given it's already part of the state, which is in scope? Same for \rho.

Reply:
This is needed for the

```julia
    Operators.column_reduce!(
        (nt1, nt2) ->
            abs(specific(nt1.ρq_tot, nt1.ρ) - q_tot_isoline) <
            abs(specific(nt2.ρq_tot, nt2.ρ) - q_tot_isoline) ? nt1 : nt2,
        isoline_z_ρ_ρq,
        Base.broadcasted(NT ∘ tuple, ᶜz, Y.c.ρ, Y.c.ρq_tot),
    )
```
call below.

Reply:
The reason that the NamedTuple is needed is that `column_reduce!` computes the output, `isoline_z_ρ_ρq`, in-place. `Y.c.ρ` and `Y.c.ρq_tot` are input arguments, and `nt1` / `nt2` are output entry values at each point along the vertical column.

Reply:
Thank you, @charleskawczynski!

Reply:
Could you please add a short docstring explaining this?